docs/source/configuration.rst

```diff
@@ -253,15 +253,27 @@ Flood Prevention
 ----------------
 
 In order to avoid flooding the server, Sopel has a built-in flood prevention
-mechanism. It can be controlled with several directives:
+mechanism. The flood burst limit can be controlled with these directives:
 
 * :attr:`~CoreSection.flood_burst_lines`: the number of messages
   that can be sent before triggering the throttle mechanism.
-* :attr:`~CoreSection.flood_empty_wait`: time to wait once burst limit has been
-  reached before sending a new message.
 * :attr:`~CoreSection.flood_refill_rate`: how much time (in seconds) must be
   spent before recovering flood limit.
 
+The wait time when the flood limit is reached can be controlled with these:
+
+* :attr:`~CoreSection.flood_empty_wait`: time to wait once burst limit has been
+  reached before sending a new message.
+* :attr:`~CoreSection.flood_max_wait`: maximum time to wait once burst limit
+  has been reached before sending a new message. This limit includes the extra
+  penalty for longer messages.
+
+And the extra wait penalty for longer messages can be controlled with these:
+
+* :attr:`~CoreSection.flood_text_length`: maximum size of messages before they
+  start getting an extra wait penalty
+* :attr:`~CoreSection.flood_penalty_ratio`: ratio used to compute said penalty
```

Comment:
```suggestion
* :attr:`~CoreSection.flood_text_length`: maximum size of messages before they
  start getting an extra wait penalty.
* :attr:`~CoreSection.flood_penalty_ratio`: ratio used to compute said penalty.
```

The preceding two list blocks have periods at the ends, so this one should too. 😛